### PR TITLE
simple calculator final

### PR DESCRIPTION
**Pull request by kayachakmak**

### CODE REVIEW

File: JS/js-variables-and-numbers_simple-calculator/js/index.js [hunk 1]
```diff
@@ -61,24 +74,42 @@ const divideByTwoButton = document.querySelector('[data-js="divide-by-two"]');
...
   // 7. Increase the value of "operand1" by 1. Log the new value to the console.
+  let operand1Upd = 12;
+  operand1Upd++;
+  console.log(operand1Upd);
 });
 
 increaseByFiveButton.addEventListener("click", () => {
   // 8. Increase the value of "operand1" by 5. Log the new value to the console.
+  let operand1Upd = 12;
+  operand1Upd += 5;
```
Comment (mgczarnievicz) [inline JS/js-variables-and-numbers_simple-calculator/js/index.js]: We are asking to do it with operand1 that is declare in line 3. Not to declare a new variable.

**Review by mgczarnievicz — CHANGES_REQUESTED** [review]
From line 52, the challenge does not do what is requier.

**Review by mgczarnievicz — CHANGES_REQUESTED** [review]
You are still doing something else. The idea is to start changing the value of `operand1` not to create a new variable.

PS. you forgot to ignore the file `.vscode`

**Comment by kayachakmak** [comment]
@mgczarnievicz I clearly misunderstood the second part of the challange.  I thought the operand1 would always have value of 12.

**Pull request merged**